phoenix-core/src/it/java/org/apache/phoenix/end2end/SystemTablesUpgradeIT.java

```diff
@@ -25,61 +25,68 @@
 import java.util.Map;
 import java.util.Properties;
 
+import org.apache.hadoop.hbase.TableName;
+import org.apache.hadoop.hbase.client.Admin;
 import org.apache.phoenix.coprocessor.MetaDataProtocol;
 import org.apache.phoenix.jdbc.PhoenixConnection;
+import org.apache.phoenix.jdbc.PhoenixDatabaseMetaData;
 import org.apache.phoenix.jdbc.PhoenixEmbeddedDriver.ConnectionInfo;
 import org.apache.phoenix.jdbc.PhoenixTestDriver;
 import org.apache.phoenix.query.BaseTest;
 import org.apache.phoenix.query.ConnectionQueryServices;
 import org.apache.phoenix.query.ConnectionQueryServicesImpl;
 import org.apache.phoenix.query.QueryServices;
 import org.apache.phoenix.query.QueryServicesTestImpl;
+import org.apache.phoenix.schema.SystemTaskSplitPolicy;
 import org.apache.phoenix.util.ReadOnlyProps;
 import org.junit.BeforeClass;
 import org.junit.Test;
 
 import org.apache.phoenix.thirdparty.com.google.common.collect.Maps;
 
-public class SystemCatalogUpgradeIT extends BaseTest {
+/**
+ * Tests for upgrades of System tables.
+ */
+public class SystemTablesUpgradeIT extends BaseTest {
     private static boolean reinitialize;
     private static int countUpgradeAttempts;
     private static long systemTableVersion = MetaDataProtocol.getPriorVersion();
-    
+
     private static class PhoenixUpgradeCountingServices extends ConnectionQueryServicesImpl {
         public PhoenixUpgradeCountingServices(QueryServices services, ConnectionInfo connectionInfo, Properties info) {
             super(services, connectionInfo, info);
         }
-        
+
         @Override
         protected void setUpgradeRequired() {
             super.setUpgradeRequired();
             countUpgradeAttempts++;
         }
-        
+
```

Comment:
These changes are specific only to whitespace issues that precommit build reported.